rustup-init.ps1

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env powershell
+
+# This is just a little script that can be downloaded from the internet to
+# install rustup. It just does platform detection, downloads the installer and
+# runs it.
+
+#XXX: If you change anything here, please make the same changes in setup_mode.rs
+function Write-Usage {
+    Write-Host @'
+rustup-init 1.18.3 (435397f48 2019-05-22)
+The installer for rustup
+
+USAGE:
+    rustup-init.exe [FLAGS] [OPTIONS]
+
+FLAGS:
+    -v, --verbose           Enable verbose output
+    -y                      Disable confirmation prompt.
+        --no-modify-path    Don't configure the PATH environment variable
+    -h, --help              Prints help information
+    -V, --version           Prints version information
+
+OPTIONS:
+        --default-host <default-host>              Choose a default host triple
+        --default-toolchain <default-toolchain>    Choose a default toolchain to install
```

Comment:
I will say that we're close to merging profiles support which will add an extra option here, though I *hope* that's just a text change for this script.

Reply:
Yeah, it should only be a text change here. I took this implementation cue from the `rustup-init.sh` script which intercepts calls with `-h` or `--help` and outputs this script-copied usage rather than downloading the `rustup-init` program and calling help on it. I'm guessing this was a resource/network saving move so I duplicated the spirit of it. Call it, some genuine "cargo culting" 😄

Reply:
Okay, that works for me.